junit-jupiter-api/junit-jupiter-api.gradle.kts

```diff
@@ -13,4 +15,20 @@ dependencies {
 	compileOnlyApi(libs.apiguardian)
 
 	compileOnly(kotlin("stdlib"))
+
+	osgiVerification(projects.junitJupiterEngine)
+	osgiVerification(projects.junitPlatformLauncher)
```

Comment:
❓ What is the `osgiVerification` configuration used for?

Reply:
This configuration is used to add dependencies to the artifacts visible to the resolver, in this case in order to calculate a complete closure of "bundles" needed a runtime we need providers for the new requirements.

Reply:
Further, this is specially define in your environment to enable the ability to tweak the OSGiVerification task without polluting any other aspects of the project.